### ISSUE TITLE
Remove redudant folders from Git project

### ISSUE THREAD

**Issue by LuiggiTenorioK** [BSC-ES/autosubmit-api#3]
In GitLab by @ jberlin on Jul 29, 2022, 17:39

As it was pointed out for the GUI in this [issue](https://earth.bsc.es/gitlab/es/autosubmitreact/-/issues/18), we should do the same analysis in the master branch API, all the redundant folders should be removed:

https://earth.bsc.es/gitlab/es/autosubmit_api/-/tree/main/dist
https://earth.bsc.es/gitlab/es/autosubmit_api/-/tree/main/build/lib/autosubmit_api

@mcastril @ggcr

**Comment by LuiggiTenorioK** [comment]
marked this issue as related to #33

**Comment by LuiggiTenorioK** [comment]
Already addressed this issue on commit 7e33d703e3381632d624817a0da1388d7842d6e5 by adding a `.gitignore` file and running:

```
git rm --cached `git ls-files -i -c --exclude-from=.gitignore` 
```
